.buildkite/scripts/build_pr_teardown.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
```

Comment:
Given the similarities between the setup and teardown scripts, does it make sense to put the bulk of the logic in a separate file which can be included and called from both?